db/schema.rb

```diff
@@ -75,18 +73,6 @@
     t.datetime "updated_at", null: false
   end
 
-  create_table "user_profiles", force: :cascade do |t|
```

Comment:
This is supposed to disappear. It got accidentally committed from a branch I was working on I think, but that feature isn't in master yet, so shouldn't be here.